clang/lib/CodeGen/CGBuiltin.cpp

```diff
@@ -513,12 +514,17 @@ static CallInst *CreateBuiltinCallWithAttr(CodeGenFunction &CGF, StringRef Name,
   // TODO: Replace AttrList with a single attribute. The call can only have a
   // single FPAccuracy attribute.
   llvm::AttributeList AttrList;
+  // "sycl_used_aspects" metadata associated with the call.
+  SmallVector<llvm::Metadata *, 4> AspectsMD;
```

Comment:
Does this need to be a vector? I am not sure if we can multiple such MD associated with a single call. Please note the TODOassociated with the AttrList above. 

Thanks